package.json

```diff
@@ -22,40 +22,41 @@
     "tslib": "^1.10.0"
   },
   "peerDependencies": {
-    "react": "^16.11.0"
+    "react": "^16.12.0"
   },
   "devDependencies": {
-    "@types/enzyme": "3.10.3",
+    "@types/enzyme": "3.10.4",
     "@types/enzyme-adapter-react-16": "1.0.5",
-    "@types/jasmine": "3.4.4",
-    "@types/lodash": "4.14.144",
-    "@types/react": "16.9.9",
-    "@types/react-dom": "16.9.2",
-    "@typescript-eslint/eslint-plugin": "2.5.0",
-    "@typescript-eslint/parser": "2.5.0",
-    "enzyme": "3.10.0",
-    "enzyme-adapter-react-16": "1.15.1",
-    "eslint": "6.6.0",
-    "eslint-config-skype": "0.5.0",
-    "eslint-plugin-react": "7.16.0",
-    "fork-ts-checker-webpack-plugin": "1.6.0",
+    "@types/jasmine": "3.5.2",
+    "@types/lodash": "4.14.149",
+    "@types/react": "16.9.19",
+    "@types/react-dom": "16.9.5",
+    "@typescript-eslint/eslint-plugin": "2.19.0",
+    "@typescript-eslint/parser": "2.19.0",
+    "enzyme": "3.11.0",
+    "enzyme-adapter-react-16": "1.15.2",
+    "eslint": "6.8.0",
+    "eslint-config-skype": "1.6.0",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-react": "7.18.3",
+    "fork-ts-checker-webpack-plugin": "4.0.3",
     "jasmine": "3.5.0",
     "jasmine-core": "3.5.0",
     "karma": "4.4.1",
     "karma-chrome-launcher": "3.1.0",
-    "karma-jasmine": "2.0.1",
-    "karma-jasmine-html-reporter": "1.4.2",
+    "karma-jasmine": "3.1.0",
+    "karma-jasmine-html-reporter": "1.5.2",
     "karma-sourcemap-loader": "0.3.7",
     "karma-spec-reporter": "0.0.32",
     "karma-webpack": "4.0.2",
     "npm-run-all": "4.1.5",
-    "react": "^16.11.0",
-    "react-dom": "^16.11.0",
-    "rimraf": "3.0.0",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "rimraf": "3.0.1",
     "ts-loader": "6.2.1",
-    "tslint": "5.20.0",
-    "typescript": "3.5.3",
-    "webpack": "4.41.2"
+    "tslint": "6.0.0",
+    "typescript": "3.8.0-dev.20200201",
```

Comment:
why dev build of 3.8?
Use 3.7.5 or 3.8.0-beta

Reply:
https://github.com/microsoft/TypeScript/issues/33269 only landed a few days ago so it's not in 3.8.0-beta